cookiecutter.json

```diff
@@ -1,15 +1,29 @@
 {
   "full_name": "First Middle Last",
```

Comment:
```suggestion
  "full_name": "<YOUR FULL NAME>",
```
Maybe we should change the default here to not confuse people if they don't have a middle name. Should the defaults have a consistent form of formatting like "<EXAMPLE>"?

Reply:
Yes, I agree. That makes a lot of sense to me.